draft-ppm-protocol.md

```diff
@@ -1033,6 +1107,24 @@ privacy threshold defined by a minimum number of input shares. One possible
 solution is to increase the reporting period so more samples can be collected,
 balanced against the urgency of responding to a soft deadline.
 
+## Protocol-specific optimizations
+
+Not all PPM tasks have the same operational requirements, so the protocol is
+designed to allow implementations to reduce operational costs in certain cases.
+
+### Reducing storage requirements
```

Comment:
This would also be a good place to discuss efficient means of tracking privacy budgets.

Reply:
We already suggest that such a scheme exists:
>  each aggregator only needs to store the aggregate output share for each possible batch interval,
along with the number of times the output share was used in a batch.

I suggest we hold off on describing this until we've implemented it. This will give us a chance to validate any assumptions we're making here. If you'd like I can flag an OPEN ISSUE here.